src/runtime/components/nuxt-img.vue

```diff
@@ -60,7 +61,8 @@ export default {
         format: this.format,
         quality: this.quality,
         background: this.background,
-        fit: this.fit
+        fit: this.fit,
+        options: this.options
```

Comment:
We do not support `modifiers.options` it should be `...this.modifiers` i guess

Reply:
...this.options you mean ?